core/src/main/resources/jenkins/model/Messages.properties

```diff
@@ -76,3 +76,5 @@ ItemCategory.Uncategorized.DisplayName=Uncategorized
 ItemCategory.Uncategorized.Description=Item types that have not yet been categorized by their plugin maintainer.
 ItemCategory.StandaloneProjects.DisplayName=Standalone Projects
 ItemCategory.StandaloneProjects.Description=Create projects with a self-contained configuration and history. These projects can be at the top-level or grouped within folders.
+ItemCategory.NestedProjects.DisplayName=Nested Projects
```

Comment:
I wondered the same thing, but IIRC @gusreiber wanted `MavenModuleSet` and `MatrixProject` to be “standalone projects” not “nested projects”.